backend/src/repos/TroopsRepo.js

```diff
@@ -0,0 +1,31 @@
+import { db } from '../data/connection';
+
+class TroopsRepo {
+  async get(kingdomID) {
+    console.log('repo');
+    console.log(kingdomID);
+    const troops = await db.query(
+      `select * from kingdom_troops where kingdom_id = ?;`,
+      kingdomID
+    );
+    return troops.results;
+  }
+
+  async insert(kingdomID, level, hp, attack, defence, started_at, finished_at) {
+    const troop = await db.query(
+      `insert into kingdom_troops (kingdom_id, level, hp, attack, defence, started_at, finished_at) values(?,?,?,?,?,?,?);`,
```

Comment:
@kkeszthe we should extract the sql table field names into constants, so if the name is changed, we need to rewrite only once